cmd/autoheal/healer.go

```diff
@@ -162,7 +166,24 @@ func (h *Healer) Run(stopCh <-chan struct{}) error {
 	// Start the workers:
 	go wait.Until(h.runRulesWorker, time.Second, stopCh)
 	go wait.Until(h.runAlertsWorker, time.Second, stopCh)
-	go wait.Until(h.runActiveJobsWorker, h.config.AWX().JobStatusCheckInterval(), stopCh)
+
+	// Start job runners
+	awxRunner, err := awxrunner.NewBuilder().
+		Config(h.config.AWX()).
+		StopCh(stopCh).
+		Build()
```

Comment:
We need to check `err` here as well, otherwise if this fails but the next does't then we don't detect it.